src/test/java/api/support/fixtures/PatronGroupExamples.java

```diff
@@ -22,4 +24,8 @@ static PatronGroupBuilder faculty() {
   static PatronGroupBuilder undergrad(){
```

Comment:
I know this line hasn't changed during this work, however I noticed the formatting here and it maya as well be fixed

```suggestion
  static PatronGroupBuilder undergrad() {
```

Reply:
Ok, I will do it